MAINTAINERS.yml

```diff
@@ -1834,6 +1834,24 @@ Nuvoton Numicro Platforms:
   labels:
     - "platform: Nuvoton Numicro"
 
+Raspberry Pi Pico Platforms:
+  status: maintained
+  maintainers:
+    - soburi
+    - yonsch
```

Comment:
FYI the order here counts, the first name is what is going to be used as assignee, which is ultimately checked by the release team to decide when to merge. @yonsch you may want to swap them if you want to take that role yourself. Also I think it's normally preferred to have only one maintainer and the rest as contributors.